x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/entity_store/routes/entity_crud/delete_entity.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { buildRouteValidationWithZod } from '@kbn/zod-helpers';
+import type { IKibanaResponse, Logger } from '@kbn/core/server';
+import type { DeleteSingleEntityResponse } from '../../../../../../common/api/entity_analytics/entity_store/entities/delete_entity.gen';
+import { DeleteSingleEntityRequestParams } from '../../../../../../common/api/entity_analytics/entity_store/entities/delete_entity.gen';
+import type { EntityAnalyticsRoutesDeps } from '../../../types';
+import { API_VERSIONS, APP_ID } from '../../../../../../common/constants';
+import {
+  BadCRUDRequestError,
+  EngineNotRunningError,
+  DocumentVersionConflictError,
+  EntityNotFoundError,
+} from '../../errors';
+import { CapabilityNotEnabledError } from '../../errors/capability_not_enabled_error';
+
+export const deleteEntity = (router: EntityAnalyticsRoutesDeps['router'], logger: Logger) => {
+  router.versioned
+    .delete({
+      access: 'public',
+      path: '/api/entity_store/entities/{entityType}/{entityId}',
+      options: {
+        availability: {
+          stability: 'beta',
```

Comment:
```suggestion
        availability: {
          since: '9.3.0',
          stability: 'beta',
```
Will this be backported or added with 9.3.0? will give something like this ("added in...") in the generated docs: 
<img width="1102" height="199" alt="image" src="https://github.com/user-attachments/assets/fecffa08-d883-4311-998e-4d9d562e5a0e" />